src/urbit/app/chess.hoon

```diff
@@ -144,6 +148,17 @@
           %=  this
             challenges-received  (~(del by challenges-received) who.action)
           ==
+        %change-special-draw-preference
+          =/  game  (~(get by games) game-id.action)
+          ?~  game
+            :_  this
+            =/  err
+              "no active game with id {<game-id.action>}"
+            :~  [%give %poke-ack `~[leaf+err]]
+            ==
+          %=  this
+            games  (~(put by games) game-id.action u.game(auto-claim-special-draws setting.action))
```

Comment:
For my own sake: what's going on with the brackets in `u.game(auto-claim-special-draws setting.action)`? I don't think I've seen that before.

Reply:
It's the irregular form of [centis](https://developers.urbit.org/reference/hoon/rune/cen#-centis) to change just that one wing of `u.game`.